package.json

```
@@ -12,6 +12,7 @@
     "build": "lerna run build --stream",
     "clean": "yarn run clean:dist && npx rimraf **/node_modules",
     "clean:dist": "npx rimraf dist packages/**/dist",
+    "docs:readme": "scripts/readme.js",
```

You can also rename your file `readme` and call it like:

```suggestion
    "docs:readme": "scripts/readme",
```

In counterpart `readme` sounds too generic as filename in my opinion. At least I suggest `update-readme`. Does it make sense for you?

---

yep 👍